Followed the guide but still Cant install

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gl@4.0.4 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gl@4.0.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\robert\AppData\Roaming\npm-cache\_logs\2018-07-23T09_58_39_180Z-debug.log

Complete log:

`PS C:\Develop\AugmentedReality\arapp> npm install gl

> gl@4.0.4 install C:\Develop\AugmentedReality\arapp\node_modules\gl
> prebuild-install || node-gyp rebuild

prebuild-install WARN install No prebuilt binaries found (target=10.0.0 runtime=node arch=x64 platform=win32)

C:\Develop\AugmentedReality\arapp\node_modules\gl>if not defined npm_config_node_gyp (node "C:\Users\robert\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\robert\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  Copying C:\Develop\AugmentedReality\arapp\node_modules\gl/deps/windows/dll/x64/libEGL.dll to C:\Develop\AugmentedReal
  ity\arapp\node_modules\gl\build\Release\libEGL.dll
          1 file(s) copied.
  Copying C:\Develop\AugmentedReality\arapp\node_modules\gl/deps/windows/dll/x64/libGLESv2.dll to C:\Develop\AugmentedR
  eality\arapp\node_modules\gl\build\Release\libGLESv2.dll
          1 file(s) copied.
  bindings.cc
  webgl.cc
  procs.cc
  win_delay_load_hook.cc
..\src\webgl.cc(459): warning C4312: 'reinterpret_cast': conversion from 'GLint' to 'GLvoid *' of greater size [C:\Deve
lop\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(778): warning C4244: '*=': conversion from 'double' to 'unsigned char', possible loss of data [C:\Devel
op\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(780): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data [C:\Dev
elop\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(781): warning C4244: '*=': conversion from 'float' to 'unsigned char', possible loss of data [C:\Develo
p\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(782): warning C4244: '*=': conversion from 'float' to 'unsigned char', possible loss of data [C:\Develo
p\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(783): warning C4244: '*=': conversion from 'float' to 'unsigned char', possible loss of data [C:\Develo
p\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(790): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data [C:\Dev
elop\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(791): warning C4244: '*=': conversion from 'float' to 'int', possible loss of data [C:\Develop\Augmente
dReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(792): warning C4244: '*=': conversion from 'float' to 'int', possible loss of data [C:\Develop\Augmente
dReality\arapp\node_modules\gl\build\webgl.vcxproj]
..\src\webgl.cc(793): warning C4244: '*=': conversion from 'float' to 'int', possible loss of data [C:\Develop\Augmente
dReality\arapp\node_modules\gl\build\webgl.vcxproj]
LINK : fatal error LNK1295: '/OPT:NOREF' not compatible with '/LTCG:incremental' specification; link without '/LTCG:inc
remental' [C:\Develop\AugmentedReality\arapp\node_modules\gl\build\webgl.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\robert\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:225:12)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\tools\\NPMnodejs\\node.exe" "C:\\Users\\robert\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Develop\AugmentedReality\arapp\node_modules\gl
gyp ERR! node -v v10.0.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok
npm WARN @angular/forms@6.0.6 requires a peer of @angular/core@6.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/forms@6.0.6 requires a peer of @angular/common@6.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/forms@6.0.6 requires a peer of @angular/platform-browser@6.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/http@6.0.6 requires a peer of @angular/core@6.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/http@6.0.6 requires a peer of @angular/platform-browser@6.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/platform-browser-dynamic@5.2.11 requires a peer of @angular/core@5.2.11 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/platform-browser-dynamic@5.2.11 requires a peer of @angular/common@5.2.11 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/platform-browser-dynamic@5.2.11 requires a peer of @angular/platform-browser@5.2.11 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gl@4.0.4 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gl@4.0.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\robert\AppData\Roaming\npm-cache\_logs\2018-07-23T10_03_54_639Z-debug.log
PS C:\Develop\AugmentedReality\arapp>














`

I've got same issue (I'm using Windows 10 64Bit + VS2017)

LINK : fatal error LNK1295: '/OPT:NOREF' not compatible with
'/LTCG:incremental' specification; link without '/LTCG:inc

This error has still remain..

--- a/binding.gyp
+++ b/binding.gyp
@@ -87,6 +87,9 @@
                     'OptimizeReferences': 1, # /OPT:NOREF
                     'EnableCOMDATFolding': 1, # /OPT:NOICF
                     'LinkIncremental': 2, # /INCREMENTAL
+                    'AdditionalOptions': [
+                      '/LTCG:OFF',
+                    ]
                   }
                 },
                 'msvs_configuration_attributes':

